guides/warning-for-head-and-tail.md

```diff
@@ -13,10 +13,10 @@ This is not a breaking change, unless a package forces `-Werror`.
 
 If you don't want to change any Haskell code and want to carry on with using `Data.List.{head,tail}`:
 
-1. If you use `-Werror`, set `-Wwarn=warnings-deprecations` to downgrade this area of compiler messages from errors to warnings, so that your build can succeed.
+1. If you use `-Werror`, set `-Wwarn=x-partial` to downgrade this area of compiler messages from errors to warnings, so that your build can succeed.
 
-2. If you are annoyed by warnings, set `-Wno-warnings-deprecations` to quash this area of compiler messages. You can also put `:set -Wno-warnings-deprecations` into `.ghci` config file.
-   Note that this also disables any other custom warnings and deprecation warnings!
+2. If you are annoyed by warnings, set `-Wno-x-partial` to quash this area of compiler messages. You can also put `:set -Wno-x-partial` into `.ghci` config file.
+   Note that this also disables any other warnings in the custom `-x-partial` warning group!
```

Comment:
It might be worth mentioning that there currently are no other warnings in that group. Is there any list of custom warning categories and their warnings in `base` this could link to?

Reply:
To my knowledge there is no such standard list, but the only entry it would contain at the moment is `x-partial` for `head`/`tail`.